quickstack/manifests/controller_common.pp

```diff
@@ -893,7 +893,8 @@
          "puppet:///modules/sensu/plugins/cpu-pcnt-usage-metrics.rb",
          "puppet:///modules/sensu/plugins/disk-metrics.rb",
          "puppet:///modules/sensu/plugins/vmstat-metrics.rb",
-         "puppet:///modules/sensu/plugins/iostat-metrics.rb"
+         "puppet:///modules/sensu/plugins/iostat-metrics.rb",
+         "puppet:///modules/sensu/plugins/vm-metrics.py"
```

Comment:
Other concern is do we need to even add the plugin script in the 
https://github.com/CCI-MOC/MOCOSPpuppet/tree/master/sensu/files/plugins 
As there also we have a list of plugins run by sensu.